.gitignore

```diff
@@ -3,3 +3,4 @@
 node_modules/
 pipeline.log
 test/integration/.hlx/
+tmp
```

Comment:
`.hlx/tmp` instead?

Reply:
was just a left-over from local testing.... not relevant